components/channel_header_dropdown/mobile_channel_header_dropdown.js

```diff
@@ -25,6 +28,18 @@ export default class MobileChannelHeaderDropdown extends React.PureComponent {
         teammateStatus: PropTypes.string,
     }
 
+    getDmUser = () => {
+        const {user, channel} = this.props;
+        const isDirect = (channel.type === Constants.DM_CHANNEL);
+        let dmUser;
+        if (channel && isDirect) {
+            const dmUserId = getUserIdFromChannelName(user.id, channel.name);
+            dmUser = getUser(store.getState(), dmUserId);
+        }
+
+        return dmUser;
```

Comment:
Access to store should be at `components/channel_header_dropdown/index.js`. I suggest to [move it there](https://github.com/asaadmahmood/mattermost-webapp/blob/MM-16393/components/channel_header_dropdown/index.js#L68) and pass it to MobileChannelHeaderDropdown as props, maybe named it as `teammateIsBot`

Reply:
Done @saturninoabril.